todoapp/build.gradle

```diff
@@ -18,14 +18,14 @@ ext {
 }
 
 buildscript {
-    ext.kotlinVersion = '1.2.30'
+    ext.kotlinVersion = '1.2.71'
     ext.gradlePluginVersion = '3.1.2'
     repositories {
-        jcenter()
         google()
+        jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:$gradlePluginVersion"
+        classpath "com.android.tools.build:gradle:3.2.1"
```

Comment:
instead of hardcoding this here, why not update the gradlePluginVersion variable?

Reply:
Thanks for checking, I fixed it.  Kotlin ver1.3.10 was released, so I also updated Kotlin to the version.